nb/documentation/interactions_with_geometry/interactions_with_geometry.md

```diff
@@ -136,7 +136,7 @@ TODO: In the future (a) => rz(a) can be replaced with rz.
 
 ```JavaScript
 Arc(6)
-  .cut(Arc(3))
+  // .cut(Arc(3))
```

Comment:
Why the delete here?

Reply:
Loft doesn't handle holes with this change.

I left the comment to re-introduce it later.

Reply:
Ah right, I forgot about that. That makes sense.